docs/samples/index.md

```diff
@@ -16,6 +16,7 @@
 | Usage       | Python                                | [Using Transformers as an external PII model](python/transformers_recognizer/index.md)                                               |
 | Usage       | Python Notebook                       | [Anonymizing known values](python/Anonymizing%20known%20values.ipynb)
 | Usage       | Python Notebook                       | [Redacting text PII from DICOM images](python/example_dicom_image_redactor.ipynb)
+| Usage       | Python Notebook                       | [Annotating PII in a PDF](python/example_pdf_annotation.ipynb)
```

Comment:
Please add to the Python TOC as well: https://github.com/microsoft/presidio/blob/main/docs/samples/python/index.md

Reply:
All sorted, thanks!